test/xcribe/writter_test.exs

```diff
@@ -0,0 +1,28 @@
+defmodule Xcribe.WritterTest do
+  use ExUnit.Case, async: false
+
+  import ExUnit.CaptureIO
+
+  alias Xcribe.Writter
+
+  @output_path "/tmp/test"
+  @invalid_output_path "/root/null"
+
+  describe "write/1" do
+    test "writes given text to set output directory" do
+      Application.put_env(:xcribe, :output, @output_path)
+
+      assert capture_io(fn ->
+               assert :ok == Writter.write("sample test")
+             end) =~ "Xcribe documentation written in #{@output_path}"
```

Comment:
It would be nice verify if the file was really written.

```suggestion
             end) =~ "Xcribe documentation written in #{@output_path}"
     
     assert File.read!(@output_path) == "sample test"
```

Reply:
Oops :sweat_smile: